bms_current_readings_test.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+import math
+import bms_current_readings_analyser as bms
+
+
+class BMSCurrentReadingTest(unittest.TestCase):
+
+    def test_if_input_data_check_yields_valid_or_invalid(self):
```

Comment:
Whenever the name of a test has conjunctions like 'or', 'and', it's an opportunity to split the test.
In this case, we can make separate tests for valid and invalid. That way, we enable open-close: additional invalids can be added without affecting the valid test.